config/schemas/config.yaml

```
@@ -65,40 +71,110 @@ $defs:
         - boolean
     properties:
       chunkLimitSize:
+        title: Chunk limit size for Fluentd buffer
+        description: |
+          Events will be written into chunks until the size of chunks become `chunkLimitSize`
+
+          [`Buffering parameters`](https://docs.fluentd.org/configuration/buffer-section#buffering-parameters)
```

```suggestion
          [Buffering parameters](https://docs.fluentd.org/configuration/buffer-section#buffering-parameters)
```
Only use preformatted for things that refer to keys or values, etc. and keep regular text outside of it.

Additionally, since the links from Fluentd is that generic does it better fit within the `buffer` object, or do we think we should push the upstream ref as close to the configuration option as possible?

---

I don't really want to do that since the Fluentd config for the buffer is a single one dimensional object.

But I would be all for organising it so that they at least are grouped this way within the schema if that isn't already done.